src/Bridge/Doctrine/Orm/ItemDataProvider.php

```diff
@@ -68,7 +69,69 @@ public function getItem(string $resourceClass, $id, string $operationName = null
             throw new ResourceClassNotSupportedException();
         }
 
-        $identifierValues = explode('-', $id);
+        $identifiers = $this->normalizeIdentifiers($id, $manager, $resourceClass);
+
+        if (!$fetchData && $manager instanceof EntityManagerInterface) {
+            return $manager->getReference($resourceClass, $identifiers);
+        }
+
+        $repository = $manager->getRepository($resourceClass);
+        $queryBuilder = $repository->createQueryBuilder('o');
+
+        $this->addWhereForIdentifiers($identifiers, $queryBuilder);
+
+        foreach ($this->itemExtensions as $extension) {
+            $extension->applyToItem($queryBuilder, $resourceClass, $identifiers, $operationName);
+        }
+
+        return $queryBuilder->getQuery()->getOneOrNullResult();
+    }
+
+    /**
+     * Add where into the query when multiple identifiers are passed (composite).
+     *
+     * @param array        $identifiers
+     * @param QueryBuilder $queryBuilder
+     *
+     * Populate the query with where when needed.
```

Comment:
This is not valid PHPDoc. It assume you forgot to clean up here.

Anyway, I suggest to rephrase it like this:
> Add WHERE conditions to the query for one or more identifiers (simple or composite).